src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2681,25 +2680,6 @@ private Variables GetOrCreateNestedFunctionVariables(Variables container, Method
             ImmutableArray<PendingBranch> pendingReturns = RemoveReturns();
             RestorePending(oldPending);
 
-            var location = lambdaOrFunctionSymbol.Locations.FirstOrNone();
-            LeaveParameters(lambdaOrFunctionSymbol.Parameters, lambdaOrFunction.Syntax, location);
-
-            // Intersect the state of all branches out of the local function
```

Comment:
Was this section just a vestige of local function definite assignment?

Reply:
Yes, it appears to be.

---
In reply to: [558751047](https://github.com/dotnet/roslyn/pull/50417#discussion_r558751047) [](ancestors = 558751047)